assets/js/_main.js

```diff
@@ -111,7 +111,7 @@ $(document).ready(function(){
   }
 
   var sidebar_shown = true;
-  var sidebar_hidden_pages = ["404", "a9lh-to-b9s", "credits", "donations", "f3-(linux)", "f3x-(mac)", "faq",
+  var sidebar_hidden_pages = ["404", "a9lh-to-b9s", "credits", "donations", "f3-(linux)", "f3xswift-(mac)", "faq",
```

Comment:
Keep in mind this won't actually touch the sidebar navigation IIRC -- you have to rebuild minimal mistakes' js file (main.min.js). Can be a bit annoying to do though and we need to fix other navigation things so don't worry about it.